doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -192,6 +192,10 @@ nrf9160 samples
 
     * Updated the library to store credentials and server settings permanently on bootstrap.
 
+  * * :ref:`lwm2m_client` sample:
+
+    * Reworked the retry logic so we are able to back off to bootstrap and we don't always restart the LTE.
```

Comment:
```suggestion
    * Reworked the retry logic so that the sample can fall back to bootstrap and need not always restart the LTE connection.
```

Reply:
Hi Uma, this proposal changes the meaning a bit.
Falling back to bootstrap is one recovery method that it does not.
Another recovery is that we restart just the client, without restarting the LTE as well. So there two things are separate.